crates/sui-core/src/event_handler.rs

```diff
@@ -68,3 +88,38 @@ impl EventHandler {
         self.event_streamer.subscribe(filter)
     }
 }
+
+fn to_json_value(move_struct: SuiMoveStruct) -> Result<Value, serde_json::Error> {
```

Comment:
Again I'm wondering what is the purpose of creating this new function vs the existing stuff to create json values